include/groonga/db.h

```diff
@@ -29,13 +29,31 @@ struct _grn_db_create_optarg {
   int n_builtin_type_names;
 };
 
+/**
+ * \brief Create the database.
```

Comment:
How about using 'new' here?
```suggestion
 * \brief Create new database.
```